docs/pages/style-spec/index.md

```diff
@@ -15,16 +14,14 @@ overviewHeader:
   changelogLink: https://github.com/maplibre/maplibre-gl-js/blob/main/src/style-spec/CHANGELOG.md
 ---
 
-A Mapbox [style](https://docs.mapbox.com/help/glossary/style/) is a document that defines the visual appearance of a map: what data to draw, the order to draw it in, and how to style the data when drawing it. A style document is a [JSON](http://www.json.org/) object with specific root level and nested properties. This specification defines and describes these properties.
+A MapLibre style is a document that defines the visual appearance of a map: what data to draw, the order to draw it in, and how to style the data when drawing it. A style document is a [JSON](http://www.json.org/) object with specific root level and nested properties. This specification defines and describes these properties.
```

Comment:
I think my `main` branch just ended up a couple of commits behind. Up to the maintainers here, but this is technically incorrect. The style document is still based on the Mapbox style specification.

Reply:
If the Maplibre team's intention is to create a wholly separate fork of the spec, they should remove the style spec's npm module from this site and update the docs to reflect that.